### PR TITLE
Feature/nexteuropa 12498

### PR DESCRIPTION
**Pull request by Fefaine**

### CODE REVIEW

File: profiles/common/modules/features/nexteuropa_multilingual/src/nexteuropa_multilingual.behat.inc [hunk 1]
```diff
@@ -207,7 +207,10 @@ class NextEuropaMultilingualSubContext extends RawDrupalContext implements Drupa
     // Assign translated title field.
     $values['title_field'][$language][0]['value'] = $fields['title'];
     // Assign translated body field, if any.
-    if (isset($fields['body']) && !empty($fields['body'])) {
+    if (isset($fields['field_ne_body']) && !empty($fields['field_ne_body'])) {
+      $values['field_ne_body'][$language][0]['value'] = $fields['field_ne_body'];
+    }
+    elseif (isset($fields['body']) && !empty($fields['body'])) {
       $values['body'][$language][0]['value'] = $fields['body'];
```
Comment (gillesdeudon) [inline profiles/common/modules/features/nexteuropa_multilingual/src/nexteuropa_multilingual.behat.inc]: IMHO, when a field is not translatable, the attached language is "LANGUAGE_NONE" (und); even if the content languageis en or nl.

**Comment by gillesdeudon** [comment]
:+1:

**Comment by gillesdeudon** [comment]
👍

**Comment by voidtek** [comment]
:+1:

**Pull request merged**